.github/settings.yml

```
@@ -0,0 +1,22 @@
+repository:
+  # https://probot.github.io/apps/settings/
+  allow_merge_commit: false
+  allow_rebase_merge: true
+  allow_squash_merge: true
+  default_branch: main
+  delete_branch_on_merge: true
+  has_discussions: false
+  has_issues: true
+  has_projects: false
+  has_wiki: false
+  private: false
```

```suggestion
  has_issues: true
```

Since all settings are optional, can we leave the ones that don't correspond to an official policy up to the specific maintainers?

In particular, we should not overwrite the private setting, as this can cause serious issues.

---

My doubt here is: what does "optional" mean? Will it rely on GitHub's defaults, or use this bot defaults?

---

Optional I think will default to GitHub's defaults, but it is not explicitly documented.

Regarding the settings, I am happy to leave it up to the maintainers - it will add importance to actually defining maintainers though 😊